index.js

```diff
@@ -24,7 +24,7 @@ var scriptTagFinder = pred.AND(
 )
 
 var slashRegex = /([^/]+$)/;
-var localeRegex = /.*_(\w{2}).json/;
+var localeRegex = /.*_(.+).json$/;
```

Comment:
I'll use the second one in case we want to support language tags including subtags that contain numbers